asaps/models.py

```diff
@@ -2,29 +2,19 @@
 import csv
 import datetime
 import json
-import logging
 import os
 import time
 
 import jsonpatch
 import jsonpointer
-import structlog
-
-logger = structlog.get_logger()
```

Comment:
Keep this line here and use this logger instance through the module rather than passing a logger in as an argument to the functions in this module.